cmd/iam_apikey_revoke.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyRevokeItemOutput egoscale.RevokeAPIKeyResponse
```

Comment:
There is no need for custom output for the `revoke` command, however the same remark applies for the `--quiet` flag support.